bootloader/test_infra/README.md

```diff
@@ -2,14 +2,14 @@
 
 This crate allows you to run the unittests against the bootloader code.
 
-You should put your tests in ../tests/bootloader/bootloader_test.yul, then compile the yul with:
+You should put your tests in `../tests/bootloader/bootloader_test.yul`, then compile the yul with:
 
 ```shell
-yarn build && yarn preprocess && yarn compile-yul
+yarn build
```

Comment:
just in case, this means you'll also have to update the `zk` tool in the `zksync-era` repo

Reply:
I still have to do this

Reply:
Created a PR: https://github.com/matter-labs/zksync-era/pull/222